homework-6/script.js

```diff
@@ -0,0 +1,20 @@
+// Write a function that accepts 1 parameter and sums up with what was passed the first time, etc. All this with closures
+
+function makeSum() {
+  let num = 0;
+  return function add(value) {
+    return (num += value);
+  };
+}
+const sum = makeSum();
+
+// Using a closure, write the structureUserInfo function, which collects the user info object from the arguments.
+
+function UserInfo(name) {
```

Comment:
why do you use Uppercase naming? In javascript this naming used only for class names